code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -893,6 +893,12 @@
 	results = list(/datum/reagent/consumable/ethanol/plasmaflood = 4)
 	required_reagents = list(/datum/reagent/toxin/plasma = 1, /datum/reagent/napalm = 1, /datum/reagent/consumable/ethanol/tequila = 1, /datum/reagent/consumable/ethanol/demonsblood = 1)
 
+/datum/chemical_reaction/drink/beeburst
```

Comment:
snake_case for readability
```suggestion
/datum/chemical_reaction/drink/bee_burst
```
Needs to go into the monkestation modular folder rather than here.
Put new changes at the bottom rather than in the middle of other files.